build.sbt

```diff
@@ -25,16 +25,16 @@ scmInfo := Some(ScmInfo(
   "scm:git:git@github.com:guardian/crosswords-api-scala-client.git"
 ))
 
-pomExtra := (
+pomExtra :=
   <url>https://github.com/guardian/crosswords-api-scala-client</url>
     <developers>
       <developer>
-        <id>robertberry</id>
-        <name>Robert Berry</name>
-        <url>https://github.com/robertberry</url>
+        <id>theguardian</id>
+        <name>The Guardian</name>
+        <url>https://github.com/guardian/</url>
       </developer>
     </developers>
-  )
+
 
```

Comment:
I wouldn't mind but we're pushing it to maven so I may as well point to the right owner :)